File: Projeto_2/Projeto_2-Paulo Victor/src/carro.py
class Carro:
    def __init__(self, ano, marca, modelo, cor, placa):
        self.ano = ano
        self.marca = marca
        self.modelo = modelo
        self.__cor = cor
        self.__placa = placa

    def exibir_detalhes(self):
        return f"{self.ano} {self.marca} {self.modelo} ({self.__cor}), Placa: {self.__placa}"

    def atualizar_cor(self, nova_cor):
        self.__cor = nova_cor

    def get_placa(self):
        return self.__placa

File: Projeto_2/Projeto_2/src/carro.js
class Carro {
  #placa;

  constructor(ano, marca, modelo, cor, placa) {
    this.ano = ano;
    this.marca = marca;
    this.modelo = modelo;
    this.cor = cor;
    this.#placa = placa;
  }

  getPlaca() {
    return this.#placa;
  }

  exibirDetalhes() {
    return `Ano: ${this.ano}, Marca: ${this.marca}, Modelo: ${
      this.modelo
    }, Cor: ${this.cor}, Placa: ${this.getPlaca()}`;
  }

  atualizarCor(novaCor) {
    this.cor = novaCor;
  }
}

export default Carro;
